consensus/service/src/tx_manager/tx_manager_trait.rs

```diff
@@ -37,10 +37,10 @@ pub trait TxManager: Send {
 
     /// Forms a Block containing the transactions that correspond to the given
     /// hashes.
-    // TODO rename
+    // TODO rename since this is no longer specific to just hashes
```

Comment:
nit: there could be a github issue to track this

Reply:
Done: https://github.com/mobilecoinfoundation/mobilecoin/issues/1685